### ISSUE TITLE
Config-file ignored when using release-pr command

### ISSUE THREAD

**Issue by fire-cracker** [google-github-actions/release-please-action#798]
### TL;DR

The `google-github-actions/release-please-action@v3` action ignores the specified config-file in the release-please YAML configuration when executing the "release-pr" command, resulting in the config-file not being utilized for the release process.

### Expected behavior

The "google-github-actions/release-please-action@v3" action should recognize and utilize the specified config-file within the YAML configuration when executing the "release-pr" command. The action should adhere to the configurations defined in the config-file during the release process.

### Observed behavior

The action does not consider the specified config-file and proceeds with the release process without adhering to the configurations defined in the config-file.

### Action YAML

```yaml
permissions:
  contents: write
  pull-requests: write

name: release-please

on: workflow_dispatch

jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - name: Create Release
        uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          command: release-pr
          config-file: release-please-config.json
```

### Log output

```text
Run google-github-actions/release-please-action@v3
  with:
    release-type: node
    command: release-pr
    config-file: release-please-config.json
    token: ***
    fork: false
    clean: true
    bump-minor-pre-major: false
    bump-patch-for-minor-pre-major: false
    changelog-host: https://github.com
    github-api-url: https://api.github.com
    github-graphql-url: https://api.github.com/graphql
    monorepo-tags: false
❯ Fetching package.json from branch master
✔ Looking for latest release on branch: master with prefix: github-actions-playground
❯ Fetching merge commits on branch master with cursor: undefined
❯ Found latest release pull request: 1 version: 1.0.0
❯ Fetching releases with cursor undefined
❯ found release for github-actions-playground Version {
  major: 1,
  minor: 0,
  patch: 0,
  preRelease: undefined,
  build: undefined
}
❯ found 2 possible releases. [
  Version {
    major: 1,
    minor: 0,
    patch: 0,
    preRelease: undefined,
    build: undefined
  },
  Version {
    major: 1,
    minor: 0,
    patch: 0,
    preRelease: undefined,
    build: undefined
  }
]
✔ Building pull requests
✔ Building strategies by path
❯ .: node
✔ Collecting release commit SHAs
❯ release search depth: 400
❯ Fetching releases with cursor undefined
❯ Found release for path ., v1.0.0
❯ release for path: ., version: 1.0.0, sha: 57b873ad123bee380565dbf90a82c0efe8938031
✔ Collecting commits since all latest releases
❯ commit search depth: 500
❯ Set(1) { '57b873ad123bee380565dbf90a82c0efe8938031' }
❯ Fetching merge commits on branch master with cursor: undefined
❯ Backfilling file list for commit: e602ce5c0c64f39c26b4748be347ff77fb4eb4bf
❯ Found 2 files
❯ Backfilling file list for commit: 0161d9e36b4a90a4f013b19d657d64044c3a4795
❯ Found 1 files
❯ Backfilling file list for commit: 60e71c81cd0e46d11647159225af88dacb0d6798
❯ Found 1 files
❯ Backfilling file list for commit: b36d678de8bb171b6c0908022e9ed0c811650188
❯ Found 1 files
❯ Backfilling file list for commit: 302c623248f8deb3c27d38cd5c4395ae6e04b9c4
❯ Found 2 files
❯ Backfilling file list for commit: 8e2d7e017715e4f0570a6bf95951d28a338f1c2d
❯ Found 4 files
❯ Backfilling file list for commit: 7e4420139c329cc5c390eed3175ccb3fd0508d9f
❯ Found 1 files
✔ Splitting 1 commits by path
✔ Building candidate release pull request for path: .
❯ type: node
❯ targetBranch: master
❯ commits: 1
✔ Considering: 1 commits
❯ component: 
❯ pull request title pattern: undefined
❯ Fetching package.json from branch master
✔ Looking for open release pull requests
✔ found 1 open release pull requests.
✔ Looking for snoozed release pull requests
✔ found 0 snoozed release pull requests.
❯ Fetching package-lock.json from branch master
✔ updating from 1.0.0 to 1.0.1
❯ Fetching npm-shrinkwrap.json from branch master
⚠ file npm-shrinkwrap.json did not exist
❯ Fetching samples/package.json from branch master
⚠ file samples/package.json did not exist
❯ Fetching CHANGELOG.md from branch master
❯ Fetching package.json from branch master
⚠ file changelog.json did not exist
✔ updating from 1.0.0 to 1.0.1
⚠ file .release-please-manifest.json did not exist
❯ Fetching changelog.json from branch master
❯ Fetching .release-please-manifest.json from branch master
✔ Starting GitHub PR workflow...
✔ Successfully found branch HEAD sha "e602ce5c0c64f39c26b4748be347ff77fb4eb4bf".
✔ Skipping branch creation step...
✔ Got the latest commit tree
✔ Successfully created a tree with the desired changes with SHA ba7a9345414aefbb1a51aefdca11477eaf3f1f7f
✔ Successfully created commit. See commit at https://api.github.com/repos/fire-cracker/github-actions-playground/git/commits/115711e5bc7b70e4a04ff9b1d3d1abdbc31ed740
✔ Updating reference heads/release-please--branches--master--components--github-actions-playground to 115711e5bc7b70e4a04ff9b1d3d1abdbc31ed740
✔ Successfully updated reference release-please--branches--master--components--github-actions-playground to 115711e5bc7b70e4a04ff9b1d3d1abdbc31ed740
✔ Found existing pull request for reference fire-cracker:release-please--branches--master--components--github-actions-playground. Skipping creating a new pull request.
✔ Successfully opened pull request: 4.
```

### Additional information

When using command `manifest` or `manifest-pr`, the `release-please-config.json` is utilized however giving their proposed deprecation I am using `release-pr` instead.

**Comment by arxeiss** [comment]
@fire-cracker where you spot that `manifest` and `manifest-pr` is going to be deprecated?

But I'm facing the same issue. It was possible for me to move everything from `release-please-config.json` into workflow file. But it is a bit ugly when it comes to defining `changelog-types`.

**Comment by chingor13** [comment]
In v3, the `release-please-config.json` file is only used for `command: manifest` or `command: manifest-pr`.

In v4, we default to using the manifest config unless you set the `release-type` option. We do highly recommend using the manifest config (it is not deprecated).
